recipes/ad3/recipe/bld.bat

```diff
@@ -0,0 +1,8 @@
+cmake -G "NMake Makefiles"  -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
+nmake install
```

Comment:
Same story as for Unix. Is there some sort of `make check` that we can run (since it appears we are building libraries)?